app/views/mappings/_mapping_table.html.haml

```diff
@@ -1,16 +1,14 @@
 = check_box_tag "delete_mappings_permission", @delete_mapping_permission, @delete_mapping_permission, style: "display: none;"
 %div#concept_mappings_tables_div
   = render_alerts_container(MappingsController)
-  %table#concept_mappings_table.table-content-stripped.table-content{width: "100%", style:'word-break: break-word'}
+  %table#concept_mappings_table.table-content-stripped.table-content.table-mini
```

Comment:
need to migrate to DataTable component

Reply:
The table here is devided between `_mapping_table.html.haml` and `_show_line.html.haml`,
also it's different than the original table component that we have anyway

Reply:
Regardless of the current setup, it should be migrated to TableComponent; otherwise, there's no point in having that component.

Reply:
Table component is already used in 19 spots, this one is little special I don't see a problem of using a regular table with the css of table component, cause we don't need the fancy functions of the component like sorting, search ...

Reply:
I did something similar with notes table it is not that complicated and will harmonize the code everywhere, here is an example https://github.com/ontoportal-lirmm/bioportal_web_ui/pull/305/files#diff-fb9722bbc8d41d19fecaead56b7b32f5c17467166749ac766f45cf6f6cf7eb75

I create this https://github.com/ontoportal-lirmm/bioportal_web_ui/issues/815 to track it, no need to do it now in this PR.

Reply:
and also this issue https://github.com/ontoportal-lirmm/bioportal_web_ui/issues/816